docs/product-business-resource-planner-guide/src/main/asciidoc/optimizer-download-proc.adoc

```diff
@@ -1,23 +1,28 @@
 [id='optimizer-download-proc']
 = Download {PLANNER}
 
-{PLANNER} is production ready.
-The API is stable but backward incompatible changes can occur.
+{PLANNER} is production ready, and can be used to optimize business solutions in {CENTRAL}, in your preferred IDE, or on OpenShift.
 
 .Prerequisite
 The {PRODUCT} has been downloaded and installed from the https://access.redhat.com[Red Hat Customer Portal]. See Installing Decision Manager for more information about installing {PRODUCT}.
 //@doc-link:Add link to Installation assembly above. 
 
 .Procedure
-. Download the {PLANNER} file from the https://access.redhat.com[Red Hat Customer Portal].
-. Unzip the {PLANNER} files.
+. Download and install {PRODUCT} from the https://access.redhat.com[Red Hat Customer Portal]. See the _Getting started with decision services_ for more information about installing {PRODUCT}.
+. Download the {PLANNER} from the https://access.redhat.com[Red Hat Customer Portal].
+. Extract the {PLANNER} files to access the complete set of {PLANNER} engine JAR files. 
+. xref:optimizer-running-the-examples-proc[Run the examples].
+
+
+{PLANNER} is production ready.
```

Comment:
This is already stated in the first paragraph.

Reply:
This is removed.